### PR TITLE
Add hirte-snapshot COPR repository description

### PR DESCRIPTION
**Pull request by mwperina**
Add hirte-snapshot COPR repository description
    
1. Adds COPR build status badge to README.md
2. Adds a description how to use hirte-snapshot COPR repository to install hirte packages

Signed-off-by: Martin Perina <mperina@redhat.com>

### CODE REVIEW

**Review by pypingou — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,6 +15,22 @@ to generate systemd service configuration to run a container.
...
 ## How to contribute
 
+### Testing
+
+RPM packages if Hirte project are available on
+[hirte-snapshot](https://copr.fedorainfracloud.org/coprs/mperina/hirte-snapshot/) COPR repo. To install hirte packages
+on your system please add that repo using:
+
+```
+dnf copr enable mperina/hirte-snapshot centos-stream-9
```
Comment (pypingou) [inline README.md]: We should build for more than just CS9, Fedora as well

File: README.md [hunk 2]
```diff
@@ -13,6 +15,22 @@ to generate systemd service configuration to run a container.
 
 ## How to contribute
 
+### Testing
+
+RPM packages if Hirte project are available on
```
Comment (pypingou) [inline README.md]:
```suggestion
RPM packages for the Hirte project are available on
```

File: README.md (hunk 1 above)
Comment (mwperina) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (mwperina) [reply README.md]: Fixed

**Review by pypingou — APPROVED** [review]

**Review by engelmi — APPROVED** [review]

**Pull request merged**